### PR TITLE
CB-10688:Cordova Plugin Splashscreen Readme must have examples

### PR DESCRIPTION
**Pull request by sarangan12**
Adding examples to the splashscreen readme. 

@rakatyal @nikhilkh @dblotsky Can you review and merge this PR?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 # cordova-plugin-splashscreen
 
-This plugin displays and hides a splash screen during application launch.
+This plugin is required to work with plugins. This plugin displays and hides a splash screen during application launch.
```
Comment (dblotsky) [inline README.md]: Did you mean to say "This plugin is required to work with splash screens"?

Comment (sarangan12) [inline README.md]: Ouch... Changed it

File: README.md [hunk 2]
```diff
@@ -44,6 +44,57 @@ Report issues with this plugin on the [Apache Cordova issue tracker][Apache Cord
...
+        <splash src="res/screen/ios/Default-736h.png" width="1242" height="2208"/>
+        <splash src="res/screen/ios/Default-Landscape-736h.png" width="2208" height="1242"/>
+    </platform>
+
+    <platform name="wp8">
+        <!-- images are determined by width and height. The following are supported -->
+        <splash src="res/screen/wp8/SplashScreenImage.jpg" width="768" height="1280"/>
+    </platform>
+
+    <platform name="windows8">
```
Comment (rakatyal) [inline README.md]: Please rename it to windows.

File: README.md [hunk 3]
```diff
@@ -70,7 +121,7 @@ In your `config.xml`, you need to add the following preferences:
     <preference name="SplashMaintainAspectRatio" value="true|false" />
     <preference name="SplashShowOnlyFirstTime" value="true|false" />
 
-Where foo is the name of the splashscreen file, preferably a 9 patch file. Make sure to add your splashcreen files to your res/xml directory under the appropriate folders. The second parameter represents how long the splashscreen will appear in milliseconds. It defaults to 3000 ms. See [Icons and Splash Screens](http://cordova.apache.org/docs/en/edge/config_ref_images.md.html)
+Where foo is the name of the splashscreen file, preferably a 9 patch file. Make sure to add your splashcreen files to your res/xml directory under the appropriate folders. The second parameter represents how long the splashscreen will appear in milliseconds. It defaults to 3000 ms. See [Icons and Splash Screens][Icons and Splash Screens]
```
Comment (rakatyal) [inline README.md]: We have renamed it to 'Use Icons' so consider using the same title.

File: README.md [hunk 4]
```diff
@@ -166,9 +217,10 @@ screen is meant to be visible before your app has started, that would seem to
...
 `config.xml` will automatically `show` the splash screen immediately after your
 app launch and before it has fully started and received the `deviceready`
-event. See [Icons and Splash Screens](http://cordova.apache.org/docs/en/edge/config_ref_images.md.html)
+event. See [Icons and Splash Screens][Icons and Splash Screens]
 for more information on doing this configuration. For this reason, it is
 unlikely you need to call `navigator.splashscreen.show()` to make the splash
 screen visible for app startup.
 
 [Apache Cordova issue tracker]: https://issues.apache.org/jira/issues/?jql=project%20%3D%20CB%20AND%20status%20in%20%28Open%2C%20%22In%20Progress%22%2C%20Reopened%29%20AND%20resolution%20%3D%20Unresolved%20AND%20component%20%3D%20%22Plugin%20Splashscreen%22%20ORDER%20BY%20priority%20DESC%2C%20summary%20ASC%2C%20updatedDate%20DESC
+[Icons and Splash Screens]: http://cordova.apache.org/docs/en/edge/config_ref_images.md.html
```
Comment (rakatyal) [inline README.md]: Why do we use the edge version and not the latest?

Comment (dblotsky) [inline README.md]: The edge version doesn't exist, BTW.

**Comment by nikhilkh** [comment]
@daserge to review

File: README.md [hunk 5]
```diff
@@ -166,9 +217,10 @@ screen is meant to be visible before your app has started, that would seem to
 defeat the purpose of the splash screen.  Providing some configuration in
 `config.xml` will automatically `show` the splash screen immediately after your
 app launch and before it has fully started and received the `deviceready`
-event. See [Icons and Splash Screens](http://cordova.apache.org/docs/en/edge/config_ref_images.md.html)
+event. See [Icons and Splash Screens][Icons and Splash Screens]
```
Comment (nikhilkh) [inline README.md]: Your change adds the docs here - so you need not even reference this page - correct?

**Comment by daserge** [comment]
@sarangan12, probably we should also include `<splash src="res/screen/windows/splashscreenphone.png" width="1152" height="1920"/>` in the example configuration (according to [cordova-windows/.../prepare.js](https://github.com/apache/cordova-windows/blob/92a298a3189ce8ce9fe6973b49dbdc777835a62b/template/cordova/lib/prepare.js#L338)).

**Comment by sarangan12** [comment]
@rakatyal @nikhilkh @dblotsky @daserge Fixed your comments. Please verify this PR

File: README.md [hunk 6]
```diff
@@ -44,6 +44,53 @@ Report issues with this plugin on the [Apache Cordova issue tracker][Apache Cord
 - Windows
 - Browser
 
+## Example Configuration
+In the top-level `config.xml` file (not the one in `platforms`), add configuration elements like those specified here.
+
+Please notice that the value of the "src" attribute is relative to the project directory and not to the www directory. You can name the source image whatever you like. The internal name in the app is determined by Cordova.
+
+    <platform name="android">
```
Comment (nikhilkh) [inline README.md]: Consider using a '```xml' as below to get syntax highlighting on GitHub:
```xml
    <myxml/>
```

File: README.md [hunk 7]
```diff
@@ -44,6 +44,53 @@ Report issues with this plugin on the [Apache Cordova issue tracker][Apache Cord
...
+    <platform name="windows">
+        <!-- images are determined by width and height. The following are supported -->
+        <splash src="res/screen/windows/splashscreen.png" width="620" height="300"/>
+        <splash src="res/screen/windows/splashscreenphone.png" width="1152" height="1920"/>
+    </platform>
+
+    <platform name="blackberry10">
+        <!-- Add a rim:splash element for each resolution and locale you wish -->
+        <!-- http://developer.blackberry.com/html5/documentation/rim_splash_element.html -->
+        <rim:splash src="res/screen/windows8/splashscreen.png"/>
```
Comment (nikhilkh) [inline README.md]: 'windows8' => 'blackberry'

**Comment by nikhilkh** [comment]
Minor comments - LGTM otherwise

**Comment by sarangan12** [comment]
@nikhilkh @rakatyal Can you please review and merge this PR?

**Comment by rakatyal** [comment]
LGTM.

**Comment by rakatyal** [comment]
Why does this show a build failure?

**Comment by sarangan12** [comment]
@rakatyal Builds are passing now

**Pull request closed without merging**